docs/api/clojure/symbol_in_pictures.md

```diff
@@ -0,0 +1,76 @@
+# Symbolic Configuration and Execution in Pictures
+
+This topic explains symbolic construction and execution in pictures.
+We recommend that you also read [Symbolic API](symbol.md).
```

Comment:
about the Symbolic API prior to attempting this tutorial?

Reply:
This isn't really a tutorial - it's more of a reference. I copied this verbatim from the Scala side, so any change we make here, we can do in both places if we want to reword

Reply:
How about `We recommend that you read the [Symbolic API](symbol.md) as another useful reference?`